samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages.properties

```diff
@@ -258,8 +258,8 @@ audio_recording = Audio Recording
 averag_grading_single_submission="Average Grading" cannot be applied to assessments which allow only single submission.
 
 due_earlier_than_avaliable=The Due Date cannot be earlier than the Available Date.
-retract_earlier_than_avaliable=The Retract Date cannot be earlier than the Available Date or the Due Date.
-retract_earlier_than_due=The Retract Date cannot be earlier than the Available Date or the Due Date.
+retract_earlier_than_avaliable=The Late Submission Date cannot be earlier than the Available Date or the Due Date.
+retract_earlier_than_due=The Late Submission Date cannot be earlier than the Available Date or the Due Date.
 retract_required_with_auto_submit=The latest acceptance date must be set to force submission of saved student work after the due date. 
```

Comment:
Should this line also be changed to Late Submission Date? Not sure, just noticed the difference between en and en_GB.